src/fstab-generator/fstab-generator.c

```diff
@@ -1163,21 +1163,21 @@ static int parse_proc_cmdline_item(const char *key, const char *value, void *dat
                 if (proc_cmdline_value_missing(key, value))
                         return 0;
 
-                return free_and_strdup_warn(&arg_root_hash, value);
+                return free_and_strdup_warn(&arg_root_hash, empty_to_null(value));
 
         } else if (streq(key, "mount.usr")) {
 
                 if (proc_cmdline_value_missing(key, value))
                         return 0;
 
-                return free_and_strdup_warn(&arg_usr_what, value);
+                return free_and_strdup_warn(&arg_usr_what, empty_to_null(value));
 
         } else if (streq(key, "mount.usrfstype")) {
 
-                if (proc_cmdline_value_missing(key, value))
+                if (proc_cmdline_value_missing(key, empty_to_null(value)))
```

Comment:
Hm? I don't think this is not necessary.

Reply:
Oops